types/lodash/array/zip.d.ts

```diff
@@ -76,7 +76,7 @@ declare module "../index" {
         zip<T>(
             this: LoDashImplicitWrapper<List<T> | null | undefined>,
             ...arrays: Array<List<T> | null | undefined>
-        ): LoDashImplicitWrapper<T[][]>;
+        ): LoDashImplicitWrapper<Array<Array<T | undefined>>>;
```

Comment:
Please add `undefined` to the other overloads for `LoDashImplicitWrapper#zip`.

Reply:
Thanks, I missed those! Updated.